app/services/platform-apps/api/modules/sources.ts

```diff
@@ -91,8 +98,8 @@ export class SourcesModule extends Module {
     const source = this.getAppSourceForApp(sourceId, ctx.app.id);
 
     source.setPropertiesManagerSettings({
-      appSettings: settings
-    });
+                                          appSettings: settings
```

Comment:
This probably isn't consistent with style guidelines.

Reply:
It isn't. Fixed, thanks for noting.